backend/resources/migrations/079-add-reviewer-role.down.sql

```diff
@@ -0,0 +1,146 @@
+-- NOTE: Postgres doesn't allow directly removing a value from an ENUM
+-- type. This migration instead creates a _old ENUM type and uses that
+-- for the role column in stakeholder table, and then renames it.  The
+-- v_stakeholder_data view needs to be deleted to be able to change
+-- the enum type, which makes this migration way bigger than needed
+-- for a simple operation!
+
+-- Remove all REVIEWER roles
```

Comment:
does `REVIEWER` exist as a stakeholder_role? I double checked sql code and didn't find it

Reply:
ups, just realised that this is the migration.down sql file 😬 and the role was defined int the up